sql/core/src/test/scala/org/apache/spark/sql/QueryTest.scala

```diff
@@ -205,6 +205,21 @@ abstract class QueryTest extends PlanTest {
         planWithCaching)
   }
 
+  /**
+   * Asserts that a given [[Dataset]] will be executed using the given named cache.
+   */
+  def assertCached(query: Dataset[_], cachedName: String): Unit = {
+    val planWithCaching = query.queryExecution.withCachedData
+    val cachedTableNames = planWithCaching collect {
```

Comment:
Avoid punctuation-free syntax when the method name is not an operator.

Reply:
thx. just add the dot in front of collect and flatten method.